crates/common/tedge_config/src/system_services/managers/config.rs

```diff
@@ -22,6 +24,23 @@ pub struct InitConfig {
     pub is_active: Vec<String>,
 }
 
+#[derive(Deserialize, Debug, Eq, PartialEq)]
+#[serde(deny_unknown_fields)]
+pub struct SystemVariablesConfig {
```

Comment:
This name provides little help guessing the purpose of this struct.

```suggestion
pub struct SystemSpecificCommands {
```